data/json/itemgroups/magazines.json

```diff
@@ -21,6 +21,8 @@
       [ "hk46mag", 15 ],
       [ "hk46bigmag", 5 ],
       [ "m107a1mag", 50 ],
+      [ "as50mag", 10 ],
```

Comment:
Don't forget this one.

Reply:
ahh dang it :smile: 

thanks for remind me.